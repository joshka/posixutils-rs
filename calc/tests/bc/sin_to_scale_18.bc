a = s(0.5)
b = s(1)
c = s(1.5)
d = s(2)
e = s(2.5)
f = s(3)

scale = 18
a/1
b/1
c/1
d/1
e/1
f/1
quit
