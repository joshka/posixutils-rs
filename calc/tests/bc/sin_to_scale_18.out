0.479425538604203000
0.841470984807896506
0.997494986604054430
0.909297426825681695
0.598472144103956494
0.141120008059867222
